libpod/container_internal_linux.go

```diff
@@ -1125,16 +1125,10 @@ func (c *Container) restore(ctx context.Context, options ContainerCheckpointOpti
 		}
 		deletedFilesPath := filepath.Join(c.bundlePath(), "deleted.files")
 		if _, err := os.Stat(deletedFilesPath); err == nil {
-			deletedFilesFile, err := os.Open(deletedFilesPath)
-			if err != nil {
-				return errors.Wrapf(err, "Failed to open deleted files file %s", deletedFilesPath)
-			}
-			defer deletedFilesFile.Close()
-
 			var deletedFiles []string
-			deletedFilesJSON, err := ioutil.ReadAll(deletedFilesFile)
+			deletedFilesJSON, err := ioutil.ReadFile(deletedFilesPath)
 			if err != nil {
-				return errors.Wrapf(err, "Failed to read deleted files file %s", deletedFilesPath)
+				return errors.Wrapf(err, "Failed to read deleted files file")
```

Comment:
F in Failed needs to be lowercased... I'm actually surprised Lint didn't catch this before your PR.

Reply:
Will address by a separate commit (in this PR).

Reply:
There are other 100+ places like this :)

See the new commit

Reply:
Some of the remaining cases can be seen by running
```console
[kir@kir-rhat libpod]$ git grep -F errors. | grep -v ^vendor | grep '"[A-Z]'
```

but I don't have the courage to fix.

Reply:
Well that's awfully concerning - I thought we had a linter to catch this, but I'm betting we lost it in the move to golangci-lint.